README.rst

```diff
@@ -60,6 +60,16 @@ public_url      BROKER_PUBLIC_URL      (none)
 allowed_origins BROKER_ALLOWED_ORIGINS (none (unrestricted))
 =============== ====================== =====================
 
+**[headers] section:**
+
+=============== ==================== ================
+``config.toml`` Environment Variable Default
+=============== ==================== ================
+static_ttl      BROKER_STATIC_TTL    604800 (1 week)
+discovery_ttl   BROKER_DISCOVERY_TTL 604800 (1 week)
+keys_ttl        BROKER_KEYS_TTL      86400 (1 day)
```

Comment:
Having the key cacheable is nice, though also problematic if combined with the key rotation of https://github.com/portier/portier-broker/pull/121#issuecomment-274454789. If clients were to really respect the cache (which I doubt any client does, but let's assume) and the rotation happens during the time, validation should fail. 

At first I wanted to suggest to you to document that problematic, but since key rotation is external to the broker, I wouldn't know where would be a good place for that. So in the end I'm just leaving this as a comment here.